### PR TITLE
Update documentation for the `file.name.template` field for S3/GCS connectors

### PR DESCRIPTION
**Pull request by tomasz-sadura**
## Description
Resolves [https://github.com/redpanda-data/documentation-private/issues/<add-your-issue-number-here>](https://github.com/redpanda-data/documentation-private/issues/2525)

## Page previews
https://deploy-preview-547--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/managed-connectors/create-s3-sink-connector/

## Checks

- [ ] New feature
- [ ] Content gap
- [X] Support Follow-up
- [ ] Small fix (typos, links, copyedits, etc)

### CODE REVIEW

**Review by bochenekmartin — APPROVED** [review]

File: modules/deploy/pages/deployment-option/cloud/managed-connectors/create-gcs-connector.adoc [hunk 1]
```diff
@@ -104,6 +104,9 @@ a| The template for file names on GCS. Supports `{{ variable }}` placeholders fo
 - `timestamp:unit=yyyy\|MM\|dd\|HH` (the timestamp of the record)
 - `key` (when used, other placeholders are not substituted)
 
+There are two sets of these placeholders supported, `topic,partition,start_offset,timestamp` and `key`, and they are exclusive.
+For the former set, the `topic,partition,start_offset` are required placeholders.
```
Comment (Feediver1) [inline modules/deploy/pages/deployment-option/cloud/managed-connectors/create-gcs-connector.adoc]:
```suggestion
There are two sets of placeholders supported, `topic,partition,start_offset,timestamp` and `key`, which  are exclusive.
For the former set, `topic,partition,start_offset` is a required placeholder.
```

Comment (Feediver1) [reply modules/deploy/pages/deployment-option/cloud/managed-connectors/create-gcs-connector.adoc]: @tomasz-sadura Is the placholder name in the last sentence correct? It does not match the previous sentence. Shouldn't it read `topic,partition,start_offset,timestamp` instead?

File: modules/deploy/pages/deployment-option/cloud/managed-connectors/create-s3-sink-connector.adoc [hunk 2]
```diff
@@ -129,6 +129,9 @@ a| The template for file names on S3. Supports `{{ variable }}` placeholders for
 - `timestamp:unit=yyyy\|MM\|dd\|HH` (the timestamp of the record)
 - `key` (when used, other placeholders are not substituted)
 
+There are two sets of these placeholders supported, `topic,partition,start_offset,timestamp` and `key`, and they are exclusive.
+For the former set, the `topic,partition,start_offset` are required placeholders.
```
Comment (Feediver1) [inline modules/deploy/pages/deployment-option/cloud/managed-connectors/create-s3-sink-connector.adoc]: Sme comment here re `topic,partition,start_offset`

File: modules/deploy/pages/deployment-option/cloud/managed-connectors/create-gcs-connector.adoc (hunk 1 above)
Comment (tomasz-sadura) [reply modules/deploy/pages/deployment-option/cloud/managed-connectors/create-gcs-connector.adoc]: The thing is the set consist of `topic`, `partition`, `start_offset` and `timestamp`. From these 4 items that are in this set, 3 are required (`topic`, `partition`, `start_offset`), while the `timestamp` is optional.

Comment (tomasz-sadura) [reply modules/deploy/pages/deployment-option/cloud/managed-connectors/create-gcs-connector.adoc]: Taken your suggestion and rephrased a little bit, please check out if it's more clear now.

**Review by Feediver1 — APPROVED** [review]
lgtm

**Pull request merged**